test/mock/test-modules.js

```diff
@@ -117,3 +117,35 @@ angular.module('ncy-sample-conf', ['ncy-sample', 'ngMock']).config(function($url
     $httpBackend.when('GET', 'views/room_detail.html').respond('dummy room_detail view');
     $httpBackend.when('GET', 'views/room_form.html').respond('dummy room_form view');
 });
+
+/**
+ * Module with abstract state with footer view
+ */
+angular.module('ncy-abstract-interpolation-conf', []).config(function($stateProvider) {
```

Comment:
I think the problem this PR solve is more related to multiple views than abstract view.

Tell me if I'm wrong but it doesn't matter if the state is abstract or not (I remove the `abstract: true` line in the plunker and the problem still occurs).

I suggest naming the conf `'ncy-multiple-interpolation-conf'`here (and update the describe's name below).